crates/rspack_plugin_css/src/visitors/mod.rs

```diff
@@ -80,26 +82,41 @@ impl VisitAstPath for Analyzer<'_> {
     }
   }
 
+  fn visit_supports_condition<'ast: 'r, 'r>(
+    &mut self,
+    n: &'ast swc_core::css::ast::SupportsCondition,
+    ast_path: &mut swc_core::css::visit::AstNodePath<'r>,
+  ) {
+    self.in_support_contdition = true;
+    n.visit_children_with_path(self, ast_path);
+    self.in_support_contdition = false;
+  }
+
   fn visit_url<'ast: 'r, 'r>(
     &mut self,
     u: &'ast Url,
     ast_path: &mut AstNodePath<AstParentNodeRef<'r>>,
   ) {
     u.visit_children_with_path(self, ast_path);
 
-    let specifier = u.value.as_ref().map(|box v| match v {
-      UrlValue::Str(s) => s.value.to_string(),
-      UrlValue::Raw(r) => r.value.to_string(),
-    });
-    if let Some(specifier) = specifier {
-      let specifier = replace_module_request_prefix(specifier, self.diagnostics);
-      let dep = Box::new(CssUrlDependency::new(
-        specifier,
-        Some(u.span.into()),
-        as_parent_path(ast_path),
-      ));
-      self.deps.push(dep.clone());
-      self.code_generation_dependencies.push(dep);
+    if !self.in_support_contdition {
+      let specifier = u.value.as_ref().map(|box v| match v {
+        UrlValue::Str(s) => s.value.trim().to_string(),
+        UrlValue::Raw(r) => r.value.trim().to_string(),
+      });
+      if let Some(specifier) = specifier && !specifier.is_empty(){
+    let mut specifier = replace_module_request_prefix(specifier, self.diagnostics);
+    if specifier.contains("%") {
+      specifier = urlencoding::decode(&specifier).expect("url invalid").to_string();
```

Comment:
what if the file path contains %, for example `%/img/a.png` and `%` is an alias for `src`

Reply:
for % and trim, you can reference to https://github.com/webpack/webpack/blob/fd3c9330b30ca30b807aee3836c89632cd58021c/lib/css/CssParser.js#L49, add `normalizeUrl(specifier)` after `replace_module_request_prefix`